types/stylelint/index.d.ts

```diff
@@ -324,6 +324,16 @@ declare module 'stylelint' {
 			 * @internal
 			 */
 			createLinter: (options: LinterOptions) => InternalApi;
+			/**
+			 * Resolves the effective configuation for a given file. Resolves to
+			 * `null` if no config is found.
+			 * @param filePath The path to the file to get the config for.
+			 * @param options The options to use when creating the Stylelint instance.
+			 */
+			resolveConfig: (
+				filePath: string,
+				options: Pick<LinterOptions, 'cwd' | 'config' | 'configBasedir' | 'configFile'>,
```

Comment:
[suggestion] Is `?` necessary to making `options` optional?

```suggestion
				options?: Pick<LinterOptions, 'cwd' | 'config' | 'configBasedir' | 'configFile'>,
```

Reply:
Yes, in fact it is, good catch!